hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/BucketEndpoint.java

```diff
@@ -244,6 +244,8 @@ public Response put(@PathParam("bucket") String bucketName,
     try {
       if (aclMarker != null) {
         s3GAction = S3GAction.PUT_ACL;
+        AUDIT.logWriteSuccess(
+            buildAuditMessageForSuccess(s3GAction, getAuditParameters()));
```

Comment:
`logWriteSuccess` should be moved to the normal, non-exception code path in `putAcl`.  Otherwise we might end up logging both success and failure for failed requests.

https://github.com/apache/ozone/blob/6354406b4a06a7d13c1bde19c95c268dd6debfbd/hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/BucketEndpoint.java#L579-L580

Reply:
The config is defined in https://github.com/apache/ozone/blob/master/hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/security/acl/OzoneAclConfig.java#L42-L48

Which is used by the Rpc Client. https://github.com/apache/ozone/blob/master/hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java#L221

This is a bit awkward for 2 reasons, the actual string for configuration cannot be searched as plain test as the configuration splits it into 2 parts. Also, the defaults are set deep inside RpcClient vs being set close to the business logic in s3g. Ideally the group security setting should have been part of the Bucket Endpoint processing itself.